tests/system/action/meeting/test_import.py

```diff
@@ -1845,6 +1845,58 @@ def test_check_missing_admin_group_in_meeting(self) -> None:
             response.json["message"],
         )
 
+    def test_with_listfields_from_migration(self) -> None:
+        """ test for listFields in event.data after migration. Uses migration 0035 to create one """
+        data = self.create_request_data({
+            "motion": {
+                "5": self.get_motion_data(
+                    5,
+                    {
+                        "title": "motion/5",
+                        "referenced_in_motion_state_extension_ids": []
+                    },
+                ),
+                "6": self.get_motion_data(
+                    6,
+                    {
+                        "title": "motion/6",
+                        "state_extension": "[motion/5]",
+                        "list_of_speakers_id": 2,
+                    },
+                ),
+           },
+            "list_of_speakers": {
+                "1": {
+                    "id": 1,
+                    "meeting_id": 1,
+                    "content_object_id": "motion/5",
+                    "closed": False,
+                    "sequential_number": 1,
+                    "speaker_ids": [],
+                    "projection_ids": [],
+                },
+                "2": {
+                    "id": 2,
+                    "meeting_id": 1,
+                    "content_object_id": "motion/6",
+                    "closed": False,
+                    "sequential_number": 2,
+                    "speaker_ids": [],
+                    "projection_ids": [],
+                },
+            },
+        })
+        data["meeting"]["meeting"]["1"]["motion_ids"] = [5, 6]
+        data["meeting"]["meeting"]["1"]["list_of_speakers_ids"] = [1, 2]
+        data["meeting"]["motion_state"]["1"]["motion_ids"] = [5, 6]
+        data["meeting"]["_migration_index"] = 35
+        assert(data["meeting"]["motion"]["5"]["referenced_in_motion_state_extension_ids"] == [])
```

Comment:
unnecessary assert since you just set the data

Reply:
Sure, but very useful to understand the test

Reply:
If the test itself is not clear enough (which this one is IMO), comments should be the way to go to describe the behaviour.